pkg/remote/README.md

```diff
@@ -0,0 +1,59 @@
+# remote: push arbitrary trace level data to an influxdb instance
+
+This package has code to create a client that can be used to push events to an
+influxdb instance. It is used to collect trace data from many different nodes in
+a network. If there is no URL in the config.toml, then the underlying client is
+nil and no points will be written. The provided chainID and nodeID are used to
+tag all points. The underlying client is exposed to allow for custom writes, but
+the WritePoint method should be used for most cases, as it enforces the schema.
+
+## Usage and Schema
+
+To use this package, first create a new client using the NewClient function,
+then pass that client to the relevant components that need to push events. After
+that, you can use the WritePoint method to push events to influxdb. In the below
+example, we're pushing a point in the consensus reactor to measure exactly when
+each step of consensus is reached for each node.
+
+```go
+cs.eventCollector.WritePoint("consensus", map[string]interface{}{
+        "height": rs.Height,
+        "round":  rs.Round,
+        "step":   rs.Step,
+    })
+```
+
+Using this method enforces the typical schema, where we are tagging (aka
+indexing) each point by the chain-id and the node-id, then adding the local time
+of the creation of the event. If you need to push a custom point, you can use
```

Comment:
> then adding the local time of the creation of the event

[non-blocking][question] how does the local time get added to the event? Does local time need to be explicitly specified or is it automatically added?

Reply:
disregard. I see below

```go
	p := write.NewPoint(table, tags, fields, time.Now())
```

so the `WritePoint` implementation automatically adds the local time.